frontend/packages/wizard/src/wizard.tsx

```diff
@@ -50,6 +53,7 @@ interface WizardStepData {
 
 interface ContainerProps {
   width?: "default" | "full";
+  enableFeedback?: boolean;
```

Comment:
a nit, can we move field to `WizardProps` instead? ContainerProps may have been intended for styling-related props

Reply:
Definitely, I think I had a misunderstanding about the Wizard/Container props. Definitely better this way. Good call!